readthedocs/proxito/views/serve.py

```diff
@@ -181,13 +181,19 @@ def get(self, request, proxito_path, template_name='404.html'):
             filename=kwargs.get('filename', ''),
         )
 
+        # Always add a `/` to the filename to match our old logic:
+        # https://github.com/readthedocs/readthedocs.org/blob/4b09c7a0ab45cd894c3373f7f07bad7161e4b223/readthedocs/redirects/utils.py#L60
+        redirect_filename = filename
+        if lang_slug and version_slug:
+            redirect_filename = '/' + filename
```

Comment:
@ericholscher If I understand correctly, our previous logic is wrong and our current one is correct. Although, since we have been supporting this, we want to keep supporting it --which means that we are making the "bug" to continue working.

If that's the case, wouldn't be better to update our data in the database to prepend a `/` to those "From URL" that are missing it and completely fix this issue instead of still supporting the bug?

Reply:
all the redirects code works this way, so I think we need to be consistent in what we're passing in. Changing the DB feels really tricky and a larger refactor.